utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/json/JsonSchemaBasedParameter.scala

```diff
@@ -58,6 +58,17 @@ object JsonSchemaBasedParameter {
             paramName,
             isRequired = None
           ) // ObjectSchema doesn't use property required
+        case _: EmptySchema =>
```

Comment:
I assume that this logic is used during dry-run and it is why you have to to add this case? It is a hacky assumption. I see that this logic is used also for logic determining parameters on sink. The user will be surprised if we change the presented parameters in sink when they use the `{}` schema